worldwind/src/androidMain/kotlin/earth/worldwind/util/kgl/AndroidKgl.kt

```diff
@@ -85,8 +85,11 @@ class AndroidKgl : Kgl {
     override fun bufferData(target: Int, size: Int, sourceData: IntArray, usage: Int, offset: Int) =
         GLES20.glBufferData(target, size, IntBuffer.wrap(sourceData, offset, size / 4), usage)
 
-    override fun bufferData(target: Int, size: Int, sourceData: FloatArray, usage: Int, offset: Int) =
-        GLES20.glBufferData(target, size, FloatBuffer.wrap(sourceData, offset, size / 4), usage)
+    override fun bufferData(target: Int, size: Int, sourceData: FloatArray?, usage: Int, offset: Int) =
```

Comment:
Make all buffer data methods able t oaccept null just for consistency.

Reply:
done, also added bufferSubData for all platforms